editor/import/resource_importer_scene.cpp

```diff
@@ -279,10 +279,13 @@ static String _fixstr(const String &p_what, const String &p_str) {
 	return what;
 }
 
-static void _gen_shape_list(const Ref<Mesh> &mesh, List<Ref<Shape3D>> &r_shape_list, bool p_convex) {
+static void _gen_shape_list(const Ref<Mesh> &mesh, List<Ref<Shape3D>> &r_shape_list, bool p_convex, bool p_single = false) {
 	if (!p_convex) {
 		Ref<Shape3D> shape = mesh->create_trimesh_shape();
 		r_shape_list.push_back(shape);
+	} else if (p_single){
```

Comment:
You're missing a space here to format the braces correctly:
```suggestion
	} else if (p_single) {
```